spec/07-bugs_spec.lua

```diff
@@ -89,4 +89,66 @@ insulate("bugs of every timer | ", function ()
 
         assert.is_false(flag)
     end)
+end)
+
+insulate("should not share ngx.ctx across different timers", function ()
+    local timer = { }
+
+    randomize()
+
+    lazy_setup(function ()
+        timer = timer_module.new({
+            min_threads = 1,
+            max_threads = 2,
+        })
+
+        assert(timer:start())
+    end)
+
+    lazy_teardown(function ()
+        timer:freeze()
+        timer:destroy()
+
+        helper.wait_until(function ()
+            assert.same(1, timer_running_count())
+            return true
+        end)
+
+    end)
+
+    before_each(function ()
+        update_time()
+    end)
+
+    it("", function ()
```

Comment:
Why does it have no description? is it OK?

Reply:
It is intentional, the description in the `describe ()` is enough.